analysis/src/HistoManager.cxx

```diff
@@ -108,7 +114,8 @@ void HistoManager::DefineHistos(std::vector<std::string> histoCopyNames, std::st
             std::cout << "DefineHisto: " << h_name << std::endl;
             std::size_t found = (hist.key()).find_last_of("_");
             std::string extension = hist.key().substr(found+1);
-
+            std::string xtitty = hist.value().at("xtitle");
```

Comment:
please put this printout of the histogram xtitles inside of an if(debug_>0)

Reply:
Done